### PR TITLE
show how to connect to remote endpoints

### PR DESCRIPTION
**Pull request by geekpete**
Show users how to connect to remote endpoints early on in the examples.

### CODE REVIEW

File: Elastic.Console/README.md [hunk 1]
```diff
@@ -135,6 +135,13 @@ es twitter/_doc/1 -Pretty -Method PUT -Body @'
 '@
 ```
 
+The default endpoint is `http://localhost:9200` but to connect to a specific endpoint use the `-Uri` parameter:
```
Comment (russcam) [inline Elastic.Console/README.md]:
```suggestion
The default endpoint is `http://localhost:9200` but you can connect to any host and port
```

File: Elastic.Console/README.md [hunk 2]
```diff
@@ -135,6 +135,13 @@ es twitter/_doc/1 -Pretty -Method PUT -Body @'
...
+
+```powershell
+es -Uri http://192.168.0.1:9200 _cat/indices
```
Comment (russcam) [inline Elastic.Console/README.md]:
```suggestion
es https://example.com:9200/_cat/indices
```

**Review by russcam — CHANGES_REQUESTED** [review]
Suggested a couple of changes

**Comment by geekpete** [comment]
No probs, I've added those changes.

**Comment by russcam** [comment]
Thanks @geekpete. Just looking at where this is in the README, it might be better to put it after the line

```
Sends a request to Elasticsearch to list the indices in the cluster. es is an alias for Invoke-Elasticsearch
```

and before the PUT document example, because the following line is the description for the PUT doc example.

**Review by russcam — APPROVED** [review]
LGTM. Thanks @geekpete 👍

**Comment by geekpete** [comment]
Thanks! Merge away :)

**Pull request merged**